src/App/Model.elm

```diff
@@ -79,20 +79,22 @@ type alias Flags =
     }
 
 
-type alias Model =
-    { currentPage : Page
-    , flags : Flags
-    , pages :
-        { home :
-            { promotedArticles : WebData (List Article)
-            , promotedRecipes : WebData (List Recipe)
-            , recipes : WebData (List Recipe)
-            }
-        , articles : WebData (List Article)
-        , recipes : Dict String (WebData (List Recipe))
-        , recipe : { recipe : WebData Recipe, recipes : WebData (List Recipe) }
-        , contact : ContactForm
+type PageModel
```

Comment:
Oh I really like this. I really disliked the nested pages model before this PR, it made is super hard to do pretty much anything. 

I like your suggestion of having one property to store the current route + one property per page